_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,1292 @@
+
+  Work on a Rust implementation was [announced][swiftsync rust impl] in
+  September.
+
+{:#dahlias}
+- **DahLIAS interactive aggregate signatures:** In April, Jonas Nick, Tim Ruffing,
+  and Yannick Seurin [announced][news351 dahlias] to the Bitcoin-Dev mailing list
+  their [DahLIAS paper][dahlias paper], the first interactive
+  64-byte aggregate signature scheme compatible with the cryptographic
+  primitives already used in Bitcoin. Aggregate signatures are
+  the cryptographic requirement for [cross-input signature aggregation][topic
+  cisa] (CISA), a feature proposed for Bitcoin that could reduce the size of
+  transactions with multiple inputs, thus reducing the cost of many different
+  types of spending, [coinjoins][topic coinjoin] and [payjoins][topic payjoin]
+  included.
+
+<div markdown="1" class="callout" id="quantum">
+
+## Summary 2025: Quantum
+
+  With the increased attention on the potential for a future [quantum][topic
+  quantum resistance] computer to weaken or break the Elliptic Curve Discrete
+  Logarithm (ECDL) hardness assumption that Bitcoin relies on to prove the
+  ownership of coins, several conversations and proposals were put forward
+  throughout the year to discuss and mitigate the impact of such a development.
+
+  Clara Shikhelman and Anthony Milton [published][news357 quantum report] a
+  paper covering the impacts of quantum computing on Bitcoin and outlining
+  potential mitigation strategies.
+
+  [BIP360][] was [updated][news bip360 update] and received its BIP
+  number. This proposal has
+  garnered support both as a first step toward quantum-hardening
+  Bitcoin and an optimization for taproot use cases that do not require an
+  internal key. [Research][news365 quantum taproot] later in the year confirmed
+  the security of these taproot commitments against manipulation by quantum
+  computers. Later in the year, the proposal was renamed to P2TSH (pay to tapscript
+  hash) instead of the earlier name P2QRH (pay to quantum resistant hash),
+  reflecting its reduced scope and increased generality.
+
+  Jesse Posner [highlighted existing research][news364 quantum primatives] that
+  indicates existing Bitcoin primitives like HD wallets, [silent
+  payments][topic silent payments], key aggregation, and [threshold
+  signatures][topic threshold signature] could be compatible with some of the
+  commonly referenced quantum-resistant signature algorithms.
+
+  Augustin Cruz [proposed][news qr cruz] a BIP to destroy coins that are
+  quantum-vulnerable with certainty. Subsequently, Jameson Lopp [started a
+  discussion][news qr lopp1] of how quantum-vulnerable coins should be handled,
+  which led to several ideas ranging from letting the quantum adversary have
+  them to destroying them. Lopp later [proposed][news qr lopp2] a concrete
+  [sequence of soft forks][BIPs #1895] that Bitcoin could implement, beginning
+  long before a CRQC is developed to gradually mitigate the threat of a quantum
```

Comment:
Since it is the first mention, perhaps:

```suggestion
  long before a Cryptographically Revelant Quantum Computer (CRQC) is developed to gradually mitigate the threat of a quantum
```